### DISCUSSION TITLE
Is defaultChecked not supported by useForm?

### DISCUSSION THREAD

**Discussion by VaZark** [mantinedev/mantine#6700] [Q&A]
I'm setting up a form wizard using the use-form hook in "uncontrolled" mode. The data is pulled from the backend via the API and "defaultValue"s are respected but the "defaultChecked" doesn't seem to work. I had to add a `defaultChecked={form.getValues().has_secondary_residence}`. 

Am I missing something? Or is this intentional?

**Comment by rtivital** [comment]
`uncontrolled` mode does not trigger rerender. if you want to use subscribe to values change it is required to set `key={form.key('has_secondary_residence')}`.
